transform/surveys/019.0018.json

```diff
@@ -1,251 +1,276 @@
 {
-  "title": "Quarterly Acquisitions and Disposals of Capital Assets Survey",
-  "survey_id": "019",
-  "form_type": "0018",
-  "comments": "true",
-  "question_groups": [
-    {
-      "title": "Reporting Period",
-      "questions": [
+    "title":"Quarterly Acquisitions and Disposals of Capital Assets Survey",
```

Comment:
I don't know why, but the format seems to have changed? Now missing a space after the `"key":`